### PR TITLE
Rebased numeric filter

### PR DESCRIPTION
**Pull request by amellnik**
This is rebased #2996.  

- [x] Run script to update copyright dates
- [x] Document internationalization entries

### CODE REVIEW

**Comment by amellnik** [comment]
I'm not sure why this is now erroring given that it passed previously and there were only trivial changes in the most recent commit.  Tests are passing locally.  It looks like it might be some sort of caching issue?

**Comment by coryrylan** [comment]
I believe you will have to rebase again on master and make a couple more changes. We merged in the latest Angular 8 update and `@ViewChild` had a breaking API change documented here: 

https://angular.io/guide/static-query-migration

**Comment by amellnik** [comment]
@coryrylan Ah good point -- it's now updated to work with 8.0.

File: package-lock.json [hunk 1]
```diff
@@ -2180,7 +2180,7 @@
     },
     "@types/node": {
       "version": "8.9.5",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-8.9.5.tgz",
+      "resolved": "http://registry.npmjs.org/@types/node/-/node-8.9.5.tgz",
```
Comment (gnomeontherun) [inline package-lock.json]: Please revert package-lock changes.

Comment (coryrylan) [reply package-lock.json]: When you revert the package-lock changes you may need to merge the `golden/clr-angular.d.ts` and/or rerun `npm run golden:fix`. Sorry about that we had a couple other PRs merge in that updated the same file. Other that that it looks great!

Comment (amellnik) [reply package-lock.json]: `package-lock.json` changes are now fixed, I rerun `npm run golden:fix` and it looks like there's no changes.

Comment (amellnik) [reply package-lock.json]: Edit: also merged in the changes with golden from master.

**Comment by amellnik** [comment]
I'm not sure what's going on -- I merged in the changes from master and ran `npm run golden:fix` but I'm still getting errors related to it: https://travis-ci.com/vmware/clarity/builds/114201737#L8668-L8700

**Comment by gnomeontherun** [comment]
I know what is happening, we literally just bumped ts-api-guardian and it exports some minor differences. Either rebase again onto latest master and run `npm ci` (this requires Node 12), or we could manually edit the output after you run it locally and remove the @deprecated comments.

**Comment by amellnik** [comment]
Thanks for the tip!  It's now all caught up and tests are passing again.

**Pull request merged**